src/storage/exec/GetNeighborsNode.h

```diff
@@ -162,23 +163,15 @@ class GetNeighborsSampleNode : public GetNeighborsNode {
 
             auto edgeType = std::get<0>(sample);
             auto val = std::get<1>(sample);
+            reader = RowReaderWrapper::getEdgePropReader(planContext_->env_->schemaMan_,
+                                                         planContext_->spaceId_,
```

Comment:
Seems we don't need reset any more ？

Reply:
Good question. Previously we need `reset` is because we don't need to reallocate if `reader` is not `nullptr`. For now, since we allocate wrapper from stack, we don't need it anymore. However, there are several `reset` method in `RowReaderWrapper`, they are for convenience, see `StorageIterator.h`. The `reset` is almost same as `getReader`.